modules/test-runner/src/main/scala/scala/build/testrunner/DynamicTestRunner.scala

```diff
@@ -108,14 +108,20 @@ object DynamicTestRunner {
   def findFramework(
     classPath: Seq[Path],
     loader: ClassLoader,
-    preferredClasses: Seq[String]
+    preferredClasses: Seq[String],
+    verbosity: Int
   ): Option[Framework] = {
     val frameworkCls = classOf[Framework]
     (preferredClasses.iterator ++ listClasses(classPath, true))
       .flatMap { name =>
         val it: Iterator[Class[_]] =
           try Iterator(loader.loadClass(name))
           catch {
+            case e: NoClassDefFoundError =>
+              if (verbosity >= 2)
```

Comment:
About the verbosity, I meant not here, but when we print `No test framework found` below. If verbosity is higher than 2, we can just keep calling `sys.error` (so throwing an exception basically) rather than calling `sys.exit` (that doesn't print a stack trace). Throwing an exception should print a stacktrace, which can be handy sometimes.